config/network.go

```diff
@@ -149,14 +149,8 @@ func (n *NetworkConfig) Validate() error {
 				return fmt.Errorf("chain ID for %s network must be set", name)
 			}
 		}
-		if n.ForkConfigs != nil {
-			if _, ok := n.ForkConfigs[network]; ok {
```

Comment:
Why remove these checks?

Reply:
While using Anvil you may choose not to fork a network, and start network from empty state like geth. Hence these checks are not required